.github/workflows/main_mai-mind-map.yml

```diff
@@ -31,6 +31,10 @@ jobs:
           pnpm install
           pnpm run build:server
           cd ..
+          cd office-addin
+          npm install

```

Comment:
Please use the same tool for managing dependencies packages.

Reply:
changed to `pnpm`